include/ddsrouter/types/endpoint/Endpoint.hpp

```diff
@@ -90,6 +90,10 @@ class Endpoint
     Endpoint& operator =(
             const Endpoint& other) noexcept;
 
+    //! Equality operator
```

Comment:
Specify in comment that does not check the `active` status of objects

Reply:
:+1: